guides/common/modules/con_troubleshooting-ruby.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Troubleshooting_Ruby_{context}"]
+= Troubleshooting Ruby
+
+If the module fails to enable, it can mean an incorrect module is enabled.
+In that case, you have to resolve dependencies manually as follows:
+
+List the enabled modules:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
+
+If the Ruby 2.7 module has already been enabled, perform a module reset:
```

Comment:
```suggestion
If the Ruby 2.5 module has already been enabled, perform a module reset:
```

Reply:
(this issue is also present in the other branches and should be fixed there too!)

Reply:
https://github.com/theforeman/foreman-documentation/pull/2257